src/sparql/rm_xref_by_prefix.ru

```diff
@@ -16,5 +16,5 @@ WHERE {
     oboInOwl:hasDbXref ?xref ;
     ?p1 ?o2 .
   }
-  FILTER( STRSTARTS(str(?xref), "UMLS_ICD9CM_2005_AUI:") || STRSTARTS(str(?xref), "ICD11:") || STRSTARTS(str(?xref), "SNOMEDCT_US_") ||STRSTARTS(str(?xref), "IMDRF:"))
+  FILTER( STRSTARTS(str(?xref), "UMLS_ICD9CM_2005_AUI:") || STRSTARTS(str(?xref), "ICD11:") || STRSTARTS(str(?xref), "SNOMEDCT_US_") || STRSTARTS(str(?xref), "IMDRF:") || STRSTARTS(str(?xref), "url:") )
```

Comment:
## URI xref removals
From what I can see, this only happens for NCIT and DOID, and it only happens after prefixes have already been renamed, so if I understand correctly, it looks like any xrefs that are no longer prefixed / recognized will be removed because they are thus in a namespace we haven't declared that we care about. So this seems fine.